firefox_ui_tests/remote/security/test_safe_browsing_notification.py

```diff
@@ -0,0 +1,120 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from marionette import (
+    By,
+    expected
+)
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestSafeBrowsingNotificationBar(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.test_data = [
+            # Phishing URL info = mozmill 18Feb2015
```

Comment:
I was noting that I'd check those URLs against the mozmill test. Of course, they can be removed now.